src/include/OpenImageIO/string_view.h

```diff
@@ -446,8 +446,6 @@ class basic_string_view {
 using string_view = basic_string_view<char>;
 using wstring_view = basic_string_view<wchar_t>;
 
-
-
 // DEPRECATED name equivalence
 OIIO_DEPRECATED("Use string_view (2.3)")
```

Comment:
Please omit this change. We don't alter formatting or whitespace in code unrelated to the subject of a PR.